sql/core/src/test/resources/sql-tests/results/udf/pgSQL/udf-aggregates_part1.sql.out

```diff
@@ -262,37 +262,37 @@ NaN	NaN
 
 
 -- !query 32
-SELECT avg(udf(CAST(x AS DOUBLE))), udf(var_pop(CAST(x AS DOUBLE)))
+SELECT CAST(avg(udf(CAST(x AS DOUBLE))) AS int), CAST(udf(var_pop(CAST(x AS DOUBLE))) AS decimal(10,3))
 FROM (VALUES (100000003), (100000004), (100000006), (100000007)) v(x)
 -- !query 32 schema
-struct<avg(CAST(udf(cast(x as double)) AS DOUBLE)):double,udf(var_pop(cast(x as double))):string>
+struct<CAST(avg(CAST(udf(cast(x as double)) AS DOUBLE)) AS INT):int,CAST(udf(var_pop(cast(x as double))) AS DECIMAL(10,3)):decimal(10,3)>
 -- !query 32 output
-1.00000005E8	2.5
+100000005	2.5
 
 
 -- !query 33
-SELECT avg(udf(CAST(x AS DOUBLE))), udf(var_pop(CAST(x AS DOUBLE)))
+SELECT CAST(avg(udf(CAST(x AS DOUBLE))) AS long), CAST(udf(var_pop(CAST(x AS DOUBLE))) AS decimal(10,3))
 FROM (VALUES (7000000000005), (7000000000007)) v(x)
 -- !query 33 schema
-struct<avg(CAST(udf(cast(x as double)) AS DOUBLE)):double,udf(var_pop(cast(x as double))):string>
+struct<CAST(avg(CAST(udf(cast(x as double)) AS DOUBLE)) AS BIGINT):bigint,CAST(udf(var_pop(cast(x as double))) AS DECIMAL(10,3)):decimal(10,3)>
 -- !query 33 output
-7.000000000006E12	1.0
+7000000000006	1
```

Comment:
Amazingly, this still fails with `INT` type. And, Spark seems to return a wrong value due to truncation.

- https://amplab.cs.berkeley.edu/jenkins/view/Spark%20QA%20Test%20(Dashboard)/job/spark-master-test-maven-hadoop-2.7/6584/testReport/org.apache.spark.sql/SQLQueryTestSuite/udf_pgSQL_udf_aggregates_part1_sql___Regular_Python_UDF/

```
Expected "700000000000[6] 1", but got "700000000000[5] 1" Result did not match for query #33&#010;SELECT CAST(avg(udf(CAST(x AS DOUBLE))) AS long), CAST(udf(var_pop(CAST(x AS DOUBLE))) AS decimal(10,3))&#010;FROM (VALUES (7000000000005), (7000000000007)) v(x)
```

Reply:
In `master` branch, all maven Jenkins fail with this. Only SBT Jenkins are running.

Reply:
BTW, in SBT Jenkins result, we can distinguish the test file name. This is a separate issue.
- https://amplab.cs.berkeley.edu/jenkins/view/Spark%20QA%20Test%20(Dashboard)/job/spark-master-test-sbt-hadoop-3.2/87/testReport/org.apache.spark.sql/SQLQueryTestSuite/

Reply:
Thanks @dongjoon-hyun. Let me take a look and make a fix soon.

Reply:
Thank you!

Reply:
It passes with Maven in my local:

```
- udf/pgSQL/udf-aggregates_part1.sql - Scala UDF
- udf/pgSQL/udf-aggregates_part1.sql - Regular Python UDF
- udf/pgSQL/udf-aggregates_part1.sql - Scalar Pandas UDF
```

I believe the problem seems similarly the Python or OS installed in the machine.

Looks here's what's going on:

```
scala> Seq("7000000000004.999", "7000000000006.999").toDF().selectExpr("CAST(avg(value) AS long)").show()
+--------------------------+
|CAST(avg(value) AS BIGINT)|
+--------------------------+
|             7000000000005|
+--------------------------+
```

Let me try to find a better way to work around it ...

Reply:
Can we ignore this test first because this is consistent and permanent in Jenkins Maven? This will hide another PR's bugs.

Reply:
Can you give me a couple of hours? I will disable this line if I can't make it ..

Reply:
Okay, let me skip this line first.

Reply:
Actually let me revert this and make another fix again. It's at least not going to block other PRs.